Runtime/Scripts/RGBotConfigs/RGState.cs

```diff
@@ -45,11 +45,10 @@ public RGStateEntity GetGameObjectState()
                 ["type"] = rgEntity.objectType,
                 ["isPlayer"] = rgEntity.isPlayer,
                 ["isRuntimeObject"] = rgEntity.isRuntimeObject,
+				["position"] = theTransform.position,
+				["rotation"] = theTransform.rotation,
```

Comment:
Formatting issue?

Reply:
@addisonbgross @rcornwall-reg this comment is marked as resolved but I still see the issue